shotgun_api3/shotgun.py

```diff
@@ -1005,12 +1037,7 @@ def schema_entity_read(self, project_entity=None):
 
         params = {}
 
-        if project_entity:
-            if not self.server_caps.version or self.server_caps.version < (5, 4, 4):
-                raise ShotgunError("Per project schema operations require server "\
-                                   "version 5.4.4 or higher, server is %s" % (self.server_caps.version,))
-            else:
-                params["project"] = project_entity
+        params = self._add_project_param(params, project_entity)
```

Comment:
@manneohrstrom, I'd been tasked w/ fixing this, and only saw much later that you'd fixed the `project` aspect of this...
So this is the conflict resolution, I hope you don't mind I kept "mine", since I'd already cherry-picked my `ensure_...` changes over from #79...